src/skill/getAvailMachinesIntent.py

```diff
@@ -0,0 +1,64 @@
+import boto3
+from utils import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource and table"""
+dynamodb = boto3.resource('dynamodb', region_name='us-east-2')
+table = dynamodb.Table('Users')
+
+
+def getAvailMachines(intent, session):
+
+    cardTitle = intent.get('name')
+    sessionAttributes = {}
+    shouldEndSession = False
+    
+    """Get buildingId from User table and machineType from session
+       Get available machines using API
+       If there are available machines, tell user how many
+       If not, offer to set a timer
+    """
+    if session.get('user'):
+        userId = session.get('user').get('userId')
+        response = table.get_item(Key = {'userId': userId})
+        
+        if response.get('Item'):
+            buildingId = response.get('Item').get('buildingId')
+            machineType = intent.get('slots').get('machineType').get('value')
+            
+            """Allow for user to use plurals when interacting with Alexa"""
+            if machineType=='washers': 
+                machineType = 'washer'
+            elif machineType == 'dryers':
+                machineType = 'dryer'
+            
+            """API Call /buildings/buildingId/machines?=available&machineType=machineType"""
+            machineInfo = callApi(
+                url = 'https://go3ba09va5.execute-api.us-east-2.amazonaws.com/Test/buildings/{}/machines?status=available&machineType={}'
+                .format(buildingId, machineType)
+            )
+            if machineInfo != {'error': 'Building or machines not found'}:

```

Comment:
This error generally be returned when the buildingId doesn't exist. If there are no machines for the building, it will just return an empty list. 

Also, we don't want to depend on the error message. doing `if machineInfo.get('error')` is a bit more robust.